po/pl.po

```diff
@@ -0,0 +1,441 @@
+# Polish translation for piper.
+# Copyright © 2017 the piper authors.
+# This file is distributed under the same license as the piper package.
+# Piotr Drąg <piotrdrag@gmail.com>, 2017.
+# Aviary.pl <community-poland@mozilla.org>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: piper\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-08-23 16:39+0200\n"
+"PO-Revision-Date: 2017-08-23 16:40+0200\n"
+"Last-Translator: Piotr Drąg <piotrdrag@gmail.com>\n"
+"Language-Team: Polish <community-poland@mozilla.org>\n"
+"Language: pl\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
+"|| n%100>=20) ? 1 : 2);\n"
+
+#. Translators: section header for mapping one button's click to another.
+#: piper/buttondialog.py:144
+msgid "Button mapping"
+msgstr "Mapowanie przycisku"
+
+#. Translators: section header for assigning special functions to buttons.
+#: piper/buttondialog.py:153
+msgid "Special mapping"
+msgstr "Mapowanie specjalne"
+
+#: piper/buttondialog.py:223 piper/buttonspage.py:82
+msgid "Button {} click"
+msgstr "Kliknięcie przycisku {}"
+
+#: piper/buttonspage.py:86
+msgid "Macro: {}"
+msgstr "Makro: {}"
+
+#. Translators: the button is turned disabled, e.g. off.
+#: piper/buttonspage.py:89
+msgid "Disabled"
+msgstr "Wyłączone"
+
+#. Translators: the button has an unknown function.
+#: piper/buttonspage.py:92 piper/ratbagd.py:613
+msgid "Unknown"
+msgstr "Nieznane"
+
+#: piper/buttonspage.py:100
+msgid "Configure button {}"
+msgstr "Konfiguracja przycisku {}"
+
+#: piper/mouseperspective.py:86 data/ui/ResolutionsPage.ui:209
+msgid "Resolutions"
+msgstr "Rozdzielczości"
+
+#: piper/mouseperspective.py:88
+msgid "Buttons"
+msgstr "Przyciski"
+
+#: piper/mouseperspective.py:90
+msgid "LEDs"
+msgstr "Diody LED"
+
+#: piper/mouseperspective.py:93 piper/profilerow.py:41
+msgid "Profile {}"
+msgstr "{}. profil"
+
+#: piper/ratbagd.py:604
+msgid "Left mouse button click"
+msgstr "Kliknięcie lewym przyciskiem myszy"
+
+#: piper/ratbagd.py:605
+msgid "Right mouse button click"
+msgstr "Kliknięcie prawym przyciskiem myszy"
+
+#: piper/ratbagd.py:606
+msgid "Middle mouse button click"
+msgstr "Kliknięcie środkowym przyciskiem myszy"
+
+#: piper/ratbagd.py:607
+msgid "Backward"
+msgstr "Wstecz"
+
+#: piper/ratbagd.py:608
+msgid "Forward"
+msgstr "Dalej"
+
+#: piper/ratbagd.py:614
+msgid "Doubleclick"
+msgstr "Podwójne kliknięcie"
+
+#: piper/ratbagd.py:615
+msgid "Wheel Left"
+msgstr "Kółko w lewo"
+
+#: piper/ratbagd.py:616
+msgid "Wheel Right"
+msgstr "Kółko w prawo"
+
+#: piper/ratbagd.py:617
+msgid "Wheel Up"
+msgstr "Kółko w górę"
+
+#: piper/ratbagd.py:618
+msgid "Wheel Down"
+msgstr "Kółko w dół"
+
+#: piper/ratbagd.py:619
+msgid "Ratchet Mode"
```

Comment:
Hmm.. I'm not sure how to explain this in a short one-liner for translators. In Dutch I'm almost certain that we'd also just be using the english term as we generally do for such things.